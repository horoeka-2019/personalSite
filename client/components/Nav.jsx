import React from "react";

class Nav extends React.Component {
  render() {
    return (
      <div>
        <ul className="navigation">
          <div className="nav-right">
            <li>
              <a href="About/about.html" className="burger">
                Jill Glass<br></br>Frontend Developer
              </a>
            </li>
          </div>
        </ul>
      </div>
    );
  }
}

export default Nav;
