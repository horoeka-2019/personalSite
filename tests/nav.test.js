import React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Nav from '../client/components/Nav'

describe('<Nav /> component test', () => {
  it('renders nav bar', () => {
    // Arrange
    const expected = 'Jill Glass'

    // Act
    const wrapper = mount(<Nav />)
    const actual = wrapper.text()
    // Assert
    expect(actual).toContain(expected)
  })
  it('render unchanged from previous snapshot', () => {
    const wrapper = mount(<Nav/>)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
 
})