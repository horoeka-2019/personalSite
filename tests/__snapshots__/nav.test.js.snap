// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> component test render unchanged from previous snapshot 1`] = `
<Nav>
  <div>
    <ul
      className="navigation"
    >
      <div
        className="nav-right"
      >
        <HashRouter>
          <Router
            history={
              Object {
                "action": "POP",
                "block": [Function],
                "createHref": [Function],
                "go": [Function],
                "goBack": [Function],
                "goForward": [Function],
                "length": 1,
                "listen": [Function],
                "location": Object {
                  "hash": "",
                  "pathname": "/",
                  "search": "",
                  "state": undefined,
                },
                "push": [Function],
                "replace": [Function],
              }
            }
          >
            <li>
              <Link
                className="burger"
                to="/about"
              >
                <LinkAnchor
                  className="burger"
                  href="#/about"
                  navigate={[Function]}
                >
                  <a
                    className="burger"
                    href="#/about"
                    onClick={[Function]}
                  >
                    <div>
                      <strong>
                        Jill Glass
                      </strong>
                    </div>
                    <div
                      className="italic"
                    >
                      Frontend Developer
                    </div>
                  </a>
                </LinkAnchor>
              </Link>
            </li>
          </Router>
        </HashRouter>
      </div>
    </ul>
  </div>
</Nav>
`;
